aws-lambdas/iac/terraform/main.tf

```diff
@@ -305,3 +330,70 @@ resource "aws_cloudwatch_event_target" "invoke_lambda" {
   rule = aws_cloudwatch_event_rule.lambda_cron.name
   arn  = module.cron.lambda_arn
 }
+
+################################################################################
+## elasticache-redis
+################################################################################
+
+module "vpc" {
+  count   = var.custom_ec_vpc != {} ? 0 : 1
+  source  = "cloudposse/vpc/aws"
+  version = "2.0.0"
+
+  ipv4_primary_cidr_block = "172.16.0.0/16"
+
+  context = {
+    tags = tomap({
+      "Name"    = "arc-lambda-boilerplate-vpc",
+      "Project" = "arc-lambda-boilerplate"
+    })
+  }
+}
+
+module "ec-subnets" {
```

Comment:
If you are creating new resources, you should be using the internal module for creating vpc and ec-subnets. 
* https://github.com/sourcefuse/terraform-aws-ref-arch-network